train_pendulum.py

```diff
@@ -0,0 +1,84 @@
+import random
+import numpy as np
+import matplotlib.pyplot as plt
+import discrete_pendulum
+import model_free
+
```

Comment:
Only missing learning curves for multiple values of alpha and epsilon. Which will be a slow process....